gatsby-node.js

```diff
@@ -66,6 +66,9 @@ exports.createSchemaCustomization = ({ actions, schema }) => {
         draft: {
           type: "Boolean",
         },
+        redirect_from: {
```

Comment:
This looks suspicious but that's probably the missing piece and reason why I didn't get it to work

Reply:
this just makes it always exist in the graphql query - so it will never error on a missing field

Reply:
is this the way to do it in gatsby? I wrote a completely backwards query in develop that just never errors.

Reply:
i would say you dont _need_ this, because gatsby auto populates the graphql schema

the only time you need this is if a field isnt present. so for example, if no docs had "redirect_from" in them, and i tried to query on that field, the queries would error